src/content/docs/query-your-data/explore-query-data/use-charts/nrql-predictions.mdx

```diff
@@ -37,129 +46,428 @@ For further details on pricing, please contact our sales team or refer to your O
+
+      Example query with the `USING` keyword with `SINCE` specification that will use 3 days of historical data (2 days from the `USING` and 1 day from the `SINCE`):
+      ```sql
+      FROM Transaction SELECT count(*) WHERE error IS TRUE TIMESERIES SINCE 1 day ago PREDICT USING 2 days
+      ```
 
-      <Collapser id="using" title={<>Add the <InlineCode>USING</InlineCode> keyword.</>}>
-      Use the `USING` keyword to specify the amount of historical data you want to use to train the model. For example, `PREDICT USING 1 day` uses the query window along with the data from the preceding day to train the model. Make sure the time span is an `integer unit` of time. There is a limit to how much data can be used to train a model and if you specify an amount that exceeds the limit, you will receive an error message. This limit is 3,660 data points and you can figure out how much time that is by using the time series window. So for example, if you are querying a time series with a 1 minute time window, you will only be able to specify an amount of training data of up to 3,660 minutes, which is 61 hours or approximately 2.5 days.
+      Example query with the `USING` keyword with both the `SINCE` and `UNTIL` specifications that will use 54 hours of historical data (2 days from the `USING` and 6 hours from the `SINCE` and `UNTIL`):
+      ```sql
+      FROM Transaction SELECT count(*) WHERE error IS TRUE TIMESERIES SINCE 1 day ago UNTIL 6 hours ago PREDICT USING 2 days
+      ```
 
-      Example query with the `USING` keyword that will use 2 days of historical data:
+      Example query with the `USING` keyword with the default query window that will use 49 hours of historical data (2 days from the `USING` and 1 hour from the default query window):
       ```sql
       FROM Transaction SELECT count(*) WHERE error IS TRUE TIMESERIES PREDICT USING 2 days
       ```
-      </Collapser>
+    </Collapser>
+
     </CollapserGroup>
 
-Example query with the `PREDICT` clause specifying custom hyperparameters, and adding the `BY` and `USING` keywords:
 
-```sql
-FROM Transaction SELECT count(*) WHERE error IS TRUE TIMESERIES PREDICT holtwinters(seasonality: AUTO, alpha: 0.2) BY 1 hour USING 2 hours
-```
+## Algorithm [#algorithm]
+Currently, <DNT>**NRQL predictions**</DNT> only supports the <DNT>**[Holt-Winters](https://en.wikipedia.org/wiki/Exponential_smoothing#Triple_exponential_smoothing_(Holt_Winters))**</DNT> algorithm, also known as exponential smoothing, for both seasonal and non-seasonal time series. This is a commonly used and standard algorithm for forecasting and time series predictions. For non-seasonal time series, only the level (average value) and trend (average increase or decrease) are modeled.
```

Comment:
I feel like we could omit "only" from the first sentence, it seems unnecessarily modest.